### PR TITLE
In llm-harness-evaluation, add new models and change schedule to nightly

### PR DESCRIPTION
**Pull request by pengyb2001**
## Description

add new models and change schedule to nightly in .github/workflows/llm-harness-evaluation.yml

### CODE REVIEW

**Comment by pengyb2001** [comment]
@zhentaocc please review the first commit

File: .github/workflows/llm-harness-evaluation.yml [hunk 1]
```diff
@@ -65,7 +67,7 @@ jobs:
         env:
           PR_MATRIX_MODEL_NAME: '["stablelm-3b-4e1t"]'
           PR_MATRIX_TASK: '["truthfulqa"]'
-          PR_MATRIX_PRECISION: '["mixed_fp4", "fp8"]'
+          PR_MATRIX_PRECISION: '["sym_int4", "fp8"]'
```
Comment (zhentaocc) [inline .github/workflows/llm-harness-evaluation.yml:70]: no need to modify this?

File: .github/workflows/llm-harness-evaluation.yml [hunk 2]
```diff
@@ -122,6 +124,7 @@ jobs:
       - name: Install dependencies
         shell: bash
         run: |
+          set -e
```
Comment (zhentaocc) [inline .github/workflows/llm-harness-evaluation.yml:127]: what's this for?

File: .github/workflows/llm-harness-evaluation.yml [hunk 3]
```diff
@@ -154,7 +157,11 @@ jobs:
       - name: Upgrade packages
         shell: bash
         run: |
-          pip install --upgrade transformers==4.34.0 datasets==2.14.6
+          pip install --upgrade datasets==2.14.6
+          if [[ "${{ matrix.model_name }}" == "Mistral-7B-v0.1" ]]; then
+          pip install --upgrade transformers==4.36
+          else
+          pip install --upgrade transformers==4.31
```
Comment (zhentaocc) [inline .github/workflows/llm-harness-evaluation.yml:164]: need to close if? Have you tested all your changes and make sure they work?

File: .github/workflows/llm-harness-evaluation.yml (hunk 2 above)
Comment (pengyb2001) [reply .github/workflows/llm-harness-evaluation.yml:127]: Exit script on any error

Comment (zhentaocc) [reply .github/workflows/llm-harness-evaluation.yml:127]: why? does it mean skipping this step if any failure occurs?

**Review by zhentaocc — APPROVED** [review]
LGTM, please update nightly label before merge

File: .github/workflows/llm-harness-evaluation.yml [hunk 4]
```diff
@@ -8,7 +8,7 @@ concurrency:
 # Controls when the action will run.
 on:
   schedule:
-    - cron: "00 13 * * 5" # GMT time, 13:00 GMT == 21:00 China
+    - cron: "00 13 * * *" # GMT time, 13:00 GMT == 21:00 China
```
Comment (hkvision) [inline .github/workflows/llm-harness-evaluation.yml:11]: also need to change time to midnight?

Comment (pengyb2001) [reply .github/workflows/llm-harness-evaluation.yml]: changed

File: .github/workflows/llm-harness-evaluation.yml [hunk 5]
```diff
@@ -50,10 +51,11 @@ jobs:
...
         env:
-          NIGHTLY_MATRIX_MODEL_NAME: '["stablelm-3b-4e1t","Mistral-7B-v0.1"]'
-          NIGHTLY_MATRIX_TASK: '["truthfulqa", "arc"]'
-          NIGHTLY_MATRIX_PRECISION: '["mixed_fp4", "fp8"]'
-          NIGHTLY_LABELS: '["self-hosted", "llm", "accuracy"]'
+          NIGHTLY_MATRIX_MODEL_NAME: '["Llama2-7b-guanaco-dolphin-500", "falcon-7b-instruct-with-patch", 
+                        "Mistral-7B-v0.1", "mpt-7b-chat", "Baichuan2-7B-Chat-LLaMAfied", "stablelm-3b"]'
+          NIGHTLY_MATRIX_TASK: '["arc", "truthfulqa", "winogrande"]'
+          NIGHTLY_MATRIX_PRECISION: '["sym_int4", "fp8"]'
+          NIGHTLY_LABELS: '["accuracy-nightly"]'
```
Comment (zhentaocc) [inline .github/workflows/llm-harness-evaluation.yml:58]: `NIGHTLY_LABELS: '["self-hosted", "llm", "accuracy-nightly"]'`

File: .github/workflows/llm-harness-evaluation.yml [hunk 6]
```diff
@@ -50,10 +51,11 @@ jobs:
       - name: set-nightly-env
         if: ${{github.event_name == 'schedule'}}
         env:
-          NIGHTLY_MATRIX_MODEL_NAME: '["stablelm-3b-4e1t","Mistral-7B-v0.1"]'
-          NIGHTLY_MATRIX_TASK: '["truthfulqa", "arc"]'
-          NIGHTLY_MATRIX_PRECISION: '["mixed_fp4", "fp8"]'
-          NIGHTLY_LABELS: '["self-hosted", "llm", "accuracy"]'
+          NIGHTLY_MATRIX_MODEL_NAME: '["Llama2-7b-guanaco-dolphin-500", "falcon-7b-instruct-with-patch", 
+                        "Mistral-7B-v0.1", "mpt-7b-chat", "Baichuan2-7B-Chat-LLaMAfied", "stablelm-3b"]'
```
Comment (hkvision) [inline .github/workflows/llm-harness-evaluation.yml:55]: we can remove stablelm-3b?

**Pull request merged**